javascript-web/package.json

```diff
@@ -20,6 +21,8 @@
     "typescript": "4.4.3"
   },
   "dependencies": {
+    "google-protobuf": "3.21.2",
```

Comment:
@cprice404 - I had to add these to get the js sdk to build but I am not sure if that was because I was including the dependency as `file:..` - do you have any idea if we actually need these?

Reply:
I also had to add 
```
"devDependencies": {
    "@types/google-protobuf": "3.15.6",
```
It got lost when I was committing. Not sure if it is actually needed if I am using a published package.